jenkins/pyinstaller/v2/pyinstaller-cortx-ha.spec

```diff
@@ -104,11 +104,25 @@ remote_execution =  Analysis([ha_path + '/ha/remote_execution/ssh_communicator.p
         cipher=block_cipher,
         noarchive=False)
 
+event_analyzer_d =  Analysis([ha_path + '/ha/core/event_analyzer/event_analyzerd.py'],
```

Comment:
event_analyzerd and event_analyzer_d
can we keep one name only 
event_analyzerd is looks good

Reply:
okay